components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -81,6 +81,48 @@ To access these components for an organization, select **Chef Servers > Organiza
 
 A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuration and policy distribution. A cookbook contains recipes and other files, components, or directories.
 
+The Chef Infra Server UI lets you:
```

Comment:
Revise this to drop the word "UI" .

Can users see the cookbooks for a single organization or for more than one organization?

Reply:
Single Organization